docs/fio.md

```diff
@@ -0,0 +1,299 @@
+# FIO Benchmark
+
+[FIO](https://github.com/axboe/fio) or Flexible IO Tester is a tool that would be able to simulate a given I/O workload without resorting to writing a tailored test case again and again.
+
+FIO spawns a number of threads or processes doing a particular type of I/O action as specified by the user. FIO takes a number of global parameters, each inherited by the thread unless otherwise parameters given to them overriding that setting is given. The typical use of FIO is to write a job file matching the I/O load one wants to simulate.
+
+## Running FIO Benchmark
+
+Once the operator has been installed following the instructions, one needs to modify the [cr.yaml](../resources/crds/benchmark_v1alpha1_benchmark_cr.yaml) to run either sequential, random or custom workload. For custom workload one can provide URL of http server where FIO job file is present.
+
+The FIO section in [cr.yaml](../resources/crds/benchmark_v1alpha1_benchmark_cr.yaml) would look like this.
+
+```yaml
+apiVersion: benchmark.example.com/v1alpha1
+kind: Benchmark
+metadata:
+  name: example-benchmark
+spec:
+  fio:
+    # To disable fio, set clients to 0
```

Comment:
Can you also put a configuration a default one without a custom job. Also update the `tests/test_crs/valid_fio.yaml` to reflect the changes too?

Reply:
It's already there. by default it will run seq workload, that's what is reflected in the doc as well.